pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -532,15 +532,16 @@ def cmd_connection_show(context, name, options):
     """
     Show the parameters that make up the current connection information if
     name is None. If Name exists, shows connections in connections file.
+    If name is "?" present list of all connections for selection.
     """
 
     connections = context.connections_repo
 
-    cname = get_current_connection_name(context)
+    curr_name = get_current_connection_name(context)
     # If no name arg, fallback to selection unless there is no connections file
     if not name:
-        name = cname or '?'
-        if not cname and not connections.file_exists():
+        name = curr_name or '?'
```

Comment:
This falls back to selection if no name is specified and there is no current connection. That case is not described in the docstring.